eng/Packages.Data.props

```diff
@@ -27,6 +27,7 @@
     <PackageReference Update="Microsoft.ServiceFabric" Version="6.4.624" />
     <PackageReference Update="Microsoft.Spatial" Version="7.5.3" />
     <PackageReference Update="Microsoft.AspNetCore.Http" Version="2.1.22" />
+		<PackageReference Update="Microsoft.Azure.KeyVault" Version="3.0.5" />
```

Comment:
Why are we adding this?   Even for a track one library, we should not be adding new dependencies on this without a strong justification.